breacharbiter.go

```diff
@@ -443,21 +523,30 @@ secondLevelCheck:
 	// channel's retribution against the cheating counter party.
 	err = b.cfg.PublishTransaction(finalTx)
 	if err != nil {
-		brarLog.Errorf("unable to broadcast "+
-			"justice tx: %v", err)
+		brarLog.Errorf("unable to broadcast justice tx: %v", err)
```

Comment:
unable -> Unable

There are a couple other cases like this.

Reply:
lowercase seems to be preferred throughout the file